<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectionWidget</class>
 <widget class="QWidget" name="InspectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="inspectionDirEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="inspectionCombo"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Inspection</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Operator</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="inspMethodEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="inspectedLengthEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Way</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="operatorEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="ObservationTable" name="observationTable"/>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="importCheckBox">
     <property name="text">
      <string>Import</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObservationTable</class>
   <extends>QTableWidget</extends>
   <header location="global">wincan2qgep.gui.observationtable</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
